Changes

```diff
@@ -79,6 +79,9 @@ _______________
 - #13045: Emphasize caution about behaviour of custom block finalizers.
   (Nick Barnes)
 
+- #13078: Fix Format tutorial on structural boxes to match the output.
```

Comment:
The PR is now more complete than this description, maybe `update Format tutorial on structural boxes to mention alignment questions`?